docs/rnm-getting-started.md

```diff
@@ -35,6 +35,10 @@ Install the React Native for macOS packages.
 npx react-native-macos-init
 ```
 
+> [!NOTE]
+> If the above command fails on ```Invalid `Podfile` file: cannot load such file -- .../ios/node_modules/react-native-macos/scripts/react_native_pods```, then it can help to open the `.xcworkspace`-file in your `ios`-folder with Xcode and straighten out any issues found there, e.g. Signing-issues
```

Comment:
Curious what issues you ran into?

Reply:
Scrolling back in my terminal output I realize I was standing in the ios-directory due to the instructions from `npx react-native@latest init...` because I had tried with Infinite Red's Ignite starter first but since rnmacos isn't expo-compatible yet I wanted to make sure ios version worked before diving into rnmacos for the first time. 

```bash
    ...
    Run instructions for iOS:
    • cd "/Users/stefanwallin/projects/mymacapp/ios" ◀️ ◀️ ◀️ ◀️ ◀️ ◀️ ◀️ ◀️ 

    • Install Cocoapods
      • bundle install # you need to run this only once in your project.
      • bundle exec pod install
      • cd ..

    • npx react-native run-ios
    ...
```

Sorry for wasting your time!

Reply:
Not a waste of time! Just want to know what issues people have in various setups :). Example: I hadn't heard of Infinite Red's ignite starter first till now, so that was useful :)